### PR TITLE
fix predicate name to keep up with others

### PR DESCRIPTION
**Pull request by mjaow**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
/kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
fix predicate name of MaxCSIVolumeCountPred to MaxCSIVolumeCount, keep up with others
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by wgliang** [comment]
If this change is going to take effect, other places that use `MaxCSIVolumeCountPred` also need to be updated, thanks.

**Review by yastij — COMMENTED** [review]
/hold

File: pkg/scheduler/algorithm/predicates/predicates.go [hunk 1]
```diff
@@ -95,7 +95,7 @@ const (
 	// All cloudprovider specific predicates are deprecated in favour of MaxCSIVolumeCountPred.
 	MaxCinderVolumeCountPred = "MaxCinderVolumeCount"
 	// MaxCSIVolumeCountPred defines the predicate that decides how many CSI volumes should be attached
-	MaxCSIVolumeCountPred = "MaxCSIVolumeCountPred"
+	MaxCSIVolumeCountPred = "MaxCSIVolumeCount"
```
[the remark this answers is not in the document]
Comment (bsalamat) [reply pkg/scheduler/algorithm/predicates/predicates.go]: As @yastij pointed out, names of predicates can be used in scheduler policy configs and changing these names may break existing clusters that use the names.

[the remark this answers is not in the document]
Comment (mjaow) [reply pkg/scheduler/algorithm/predicates/predicates.go]: yeah,that's a problem.Can we just make the bad name deprecated and come over to new name

**Comment by mjaow** [comment]
Some of the bad names will exist forever in this case and some looks really not beaultiful. Should we make it sliding to new names smoothly？Maybe add new name and reserve old one for a while ,and finally drop the old one

**Pull request closed without merging**